site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -45,6 +46,29 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
 };
 
+const editThread = (title) => {
+    cy.get('.thread-left-cont > .thread-list-item').contains(title).click();
+    cy.get('.create-post-head').should('contain', title);
+    cy.get('.first_post > .post-action-container > .edit-post-button').click();
```

Comment:
the code which is already present before this pr

```js
// Line 20 
cy.get('.flex-row > .thread-left-cont').should('contain', title);

// Line 44
cy.get('.first_post > .post-action-container > .delete-post-button').click();
```

so i have to change mine or could also this code too (i planned to do in data-testid manner but so the previous code is in that manner so i implemented like that ) 

could i changed it or leave it ?

Reply:
This test was written before the 'testid' syntax was added as the recommendation. Anything new it would be better to add testid because its coupled to many attributes with HTML semantics